backends/v3/src/main.rs

```diff
@@ -68,10 +68,8 @@ struct Args {
     disable_grammar_support: bool,
     #[clap(default_value = "4", long, env)]
     max_client_batch_size: usize,
-    #[clap(long, env, default_value_t)]
-    disable_usage_stats: bool,
-    #[clap(long, env, default_value_t)]
-    disable_crash_reports: bool,
+    #[clap(default_value = "on", long, env)]
+    usage_stats: Option<String>,
```

Comment:
Make it a real enum instead ? Saves you writing the actual parsing logic.

Reply:
Ah yeah ofc, I missed the derive ValueEnum 👍